central/billingmetrics/store/postgres/store.go

```diff
@@ -0,0 +1,150 @@
+package postgres
+
+// Code is not generated by pg-bindings generator. DO EDIT.
```

Comment:
Why can't this code be generated by pg-bindings? It would be better if we could keep it generated

Reply:
Because generator cannot generate the required logic today.